.meta/sinks/http.toml.erb

```diff
@@ -103,6 +103,15 @@ type = "string"
 examples = ["https://10.22.212.22:9000/_health"]
 description = "A URI that Vector can request in order to determine the service health."
 
+[sinks.http.options.metrics_plus_mode]
+type = "bool"
+examples = [true, false]
+description = """\
+When set to true this sink will use a text format that supports metrics. \
+The default format only supports logs. The receiving source must have the same value\
```

Comment:
Users can already transmit metrics between vector sources and sinks. This caveat kind of limits the use of this feature.